<launch>

  <node pkg="open_abb_driver" type="abb_node" name="abb_arm" output="screen">
  </node>

</launch>
